py/server/deephaven/parquet.py

```diff
@@ -43,6 +43,7 @@ def _build_parquet_instructions(
     is_refreshing: bool = False,
     for_read: bool = True,
     force_build: bool = False,
+    generate_metadata_files: Optional[bool] = False,
```

Comment:
Are you sure this shouldn't be:
```suggestion
    generate_metadata_files: Optional[bool] = None,
```
As is, you will rarely get in the `if not any` code block, which I suspect is your target block.